internal/pkg/scaffold/input/input.go

```diff
@@ -105,6 +108,19 @@ func (i *Input) SetProjectName(n string) {
 	}
 }
 
+// OperatorName allows the operator name to be set on an object
```

Comment:
These are unnecessary once you move `OperatorName` to `scaffold.CSV`.

Reply:
done